src/messaging-component.js

```diff
@@ -51,8 +51,11 @@ const showExitModal = ({ cartRecovery }) => {
     return isRendered;
 };
 
-export const Messaging = (...args : $ReadOnlyArray<{ cartRecovery : { userId : string } }>) => {
+export const Messaging = (...args : $ReadOnlyArray<{ cartRecovery : { userId : string, beta? : boolean } }>) => {
     const config = args[0];
+    if (!config.cartRecovery || !config.cartRecovery.beta) {
```

Comment:
Also checked `config.cartRecovery` here since it seems pointless to setup the event listeners if they aren't going to be used to call anything.